packages/components/navlist/src/NavList/NavList.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { cx } from 'emotion';
+import {
+  type CommonProps,
+  type PolymorphicProps,
+  type PolymorphicComponent,
+  type ExpandProps,
+} from '@contentful/f36-core';
+import { getStyles } from './NavList.styles';
+
+const NAV_LIST_DEFAULT_TAG = 'div';
+
+export interface NavListInternalProps extends CommonProps {
+  children: React.ReactNode;
+  /**
+   * @default div
+   */
+  as?: 'nav' | 'div';
+}
+
+export type NavListProps<
+  E extends React.ElementType = typeof NAV_LIST_DEFAULT_TAG,
+> = PolymorphicProps<NavListInternalProps, E>;
+
+function _NavList<E extends React.ElementType = typeof NAV_LIST_DEFAULT_TAG>(
+  props: NavListProps<E>,
+  ref: React.Ref<any>,
+) {
+  const styles = getStyles();
+
+  const { as, className, children, testId, ...otherProps } = props;
+  const role = as === 'nav' ? 'navigation' : undefined;
```

Comment:
I think should be the other way around, when it's a nav the role is `navigation` out of the box, we only need to set role when it's a `div`

Reply:
nice catch, missed that